README.md

```diff
@@ -129,117 +129,117 @@ The behaviour of balenaOS can be configured by setting the following keys in the
+  "wifi": {
+    "randomMacAddressScan": false
   }
+ }
 }
 ```
 
 ##### connectivity
 
-This object defines configuration related to networking connectivity checks:
-   * "uri" string where the value is the url to query for connectivity checks
-   * "interval" string where the value is the interval between connectivity checks in seconds.
-   * "response" string. If set, controls what body content is checked for when requesting the URI for connectivity. If it is an empty value, the HTTP server is expected to answer with status code 204 or send no data.
+An object that defines configuration related to networking connectivity checks. This feature builds on NetworkManager's connectivity check, which is further documented in the connectivity section [here](https://developer.gnome.org/NetworkManager/stable/NetworkManager.conf.html).
 
-The default values are
-```
-uri=$API_ENDPOINT/connectivity-check
-interval=3600
-response=
-```
+- "uri" (string) Value of the url to query for connectivity checks. Defaults to `$API_ENDPOINT/connectivity-check`.
+- "interval" (string) Interval between connectivity checks in seconds. Defaults to 3600.
+- "response" (string). If set controls what body content is checked for when requesting the URI. If it is an empty value, the HTTP server is expected to answer with status code 204 or send no data.
 
-See below an example of a config.json snippet which configures the connectivity check by passing the balena cloud connectivity endpoint with a 5 minute interval.
-```
+The following example configures the connectivity check by passing the balenaCloud connectivity endpoint with a 5-minute interval.
+
+```json
 "os": {
-  "network" : {
-    "connectivity": {
-        "uri" : "https://api.balena-cloud.com/connectivity-check",
-        "interval" : "300"
-      }
+ "network" : {
+  "connectivity": {
+    "uri" : "https://api.balena-cloud.com/connectivity-check",
+    "interval" : "300"
   }
+ }
 }
 ```
 
 #### udevRules
 
-String. Custom udev rules can be passed via config.json.
+An object containing one or more custom udev rules as `key:value` pairs.
 
-To turn a rule into the format that can be easily added to config.json, use
+To turn a rule into a format that can be easily added to `config.json`, use the following command:
 
-`cat rulefilename | jq -sR .`
-e.g.
+```shell
+cat rulefilename | jq -sR .
 ```
+
+For example:
+
+```shell
 root@resin:/etc/udev/rules.d# cat 64.rules | jq -sR .
 "ACTION!=\"add|change\", GOTO=\"modeswitch_rules_end\"\nKERNEL==\"ttyACM*\", ATTRS{idVendor}==\"1546\", ATTRS{idProduct}==\"1146\", TAG+=\"systemd\", ENV{SYSTEMD_WANTS}=\"u-blox-switch@'%E{DEVNAME}'.service\"\nLBEL=\"modeswitch_rules_end\"\n"
-root@resin:/etc/udev/rules.d#
 ```
 
-An example config.json snippet with 2 rules:
+The following example contains two custom udev rules that will create `/etc/udev/rules.d/56.rules` and `/etc/udev/rules.d/64.rules`. The first time rules are added/modified, udevd will reload the rules and re-trigger.
```

Comment:
Maybe rephrase to "The first time rules are added or when they are modified, udevd will reload the rules and re-trigger."